WordPress/twentytwentyone
# Fixed focus state site logo

Fixes https://github.com/WordPress/twentytwentyone/issues/440

## Summary


That `<a>` that surrounded the image had no width in Chrome so the focus state wouldn't show up.

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Go to the customizer
2. Add a custom logo, save and leave the customizer
3. Navigate to a page other than the homepage
4. Tab to the logo link
5 .See that there is now a visible focus


## Screenshots
<img width="1279" alt="Screenshot 2020-10-13 at 11 42 29" src="https://user-images.githubusercontent.com/62988563/95844575-5b45b200-0d49-11eb-8b6e-a8f2e8c69b37.png">


## Quality assurance
